cli/cmd/sync_v3.go

```diff
@@ -748,9 +753,57 @@ func syncConnectionV3(ctx context.Context, syncOptions syncV3Options) (syncErr e
 		Str("result", msg).
 		Msg("Sync summary")
 
+	if totalResources > 0 {
+		hintSelectMessage(sourceSpec.Path, destPaths, statsPerTable)
+	}
+
 	return nil
 }
 
+func hintSelectMessage(sourcePath string, destinationPaths map[string]bool, statsPerTable *utils.ConcurrentMap[string, cloudquery_api.SyncRunTableProgressValue]) {
+	if sourcePath != "cloudquery/aws" {
+		return
+	}
+
+	val, _ := config.GetValue("first_sync_completed")
+	firstSyncCompleted, _ := strconv.ParseBool(val)
+	if firstSyncCompleted {
+		return
+	}
+
+	if auth.NewTokenClient().GetTokenType() != auth.BearerToken {
+		return
+	}
+
+	rows, ok := statsPerTable.Get("aws_ec2_instances")
+	if !ok || rows.Rows == 0 {
+		return
+	}
+
+	switch {
+	case destinationPaths["cloudquery/postgresql"]:
+		fmt.Println()
+		fmt.Println("🎉 Success!")
+		fmt.Println()
+		fmt.Println("Run the following command to get your oldest 10 EC2 instances:")
+		fmt.Println()
+		fmt.Println(`SELECT account_id, instance_id, region, launch_time FROM aws_ec2_instances ORDER BY launch_time ASC LIMIT 10`)
+	case destinationPaths["cloudquery/sqlite"]:
+		fmt.Println()
+		fmt.Println("🎉 Success!")
+		fmt.Println()
+		fmt.Println("Run the following command to get your oldest 10 EC2 instances:")
+		fmt.Println()
+		fmt.Println(`sqlite3 db.sql "SELECT account_id, instance_id, region, launch_time FROM aws_ec2_instances ORDER BY launch_time ASC LIMIT 10"`)
```

Comment:
I guess we can get the correct file path from the spec? Instead of hard coding `db.sql`